### DISCUSSION TITLE
CORS issues after redeploying from backup due to getting locked out of admin

### DISCUSSION THREAD

**Discussion by daCking15** [mozilla/hubs#2890] [Hubs Cloud Questions] [answered]
After redeploying the stack, I have access to the admin settings, but I am getting a bunch of CORS issues still. I tried updating the Allowed CORS Origins settings to include the domain, link domain, and localhost, but I am still unable to create/enter rooms and on top of that, the preview images for the scenes in the hubs cloud control panel won't load.

**Comment by robinkwilson** [comment, marked as answer]
You'll need to delete the cors_origin setting in the aws parameter store to revert back to defaults. [How to delete cors origin setting from parameter store discussion thread](https://github.com/mozilla/hubs/discussions/2863)
Everything will be working fine after that!

[OPTIONAL] If you still want to add a domain to your CORS settings, in the Admin Panel CORS origin field, use the below example to fill in your stack's domains + add the domains you want to add.

**Delete the cors_origin setting in AWS Parameter store to revert to defaults.**
1. Go to Parameter Store
   - AWS console > Systems Manager > Parameter Store (try [this shortcut link](https://console.aws.amazon.com/systems-manager/parameters))
2. Check that you're in the correct region: Are your stack parameters listed in the menu?
   - Yes, you're in the correct region
   - No, select your deploy region (upper right hand corner next to your name)
3. Select then delete this parameter: **/ita/{stack name}/reticulum/security/cors_origins**
4. Check that you have access to your Admin panel again
   - Deleting the setting in the parameter store should be enough, but you can also restart the instance:
   - AWS console > EC2 > click on your HC server(s) then "Terminate" instances. New ones will start up immediately after.
5. You're back into your hub!!

**[OPTIONAL] Fill in settings in the Admin Panel (and add additional domains if you want):**
1. Go to your hub's main domain > Admin Panel > Server Settings > Advanced tab
2. In your "Allowed CORS Origin" field:

> **https://{main domain}.com, https://{link domain}.link, {main domain}.com, {link domain}.link, https://{internal domain}.com, {internal domain}.com**

^^ Here's what worked for me. REMEMBER to change {main domain}, {internal domain}, {link domain}, and the domain extension (.com, .link) to your hub's. 
(I added the internal domain using the same standard (my stack used Recipe 1 so it didn't have a different internal domain)

**Reply by netpro2k** [reply]
Kind of surprised you need to manually set a value at all. Shouldn't deleting the parameter store value just get things to go back to using defaults? Or do normally we bake in a default into the parameter store as part of stack creation somehow?

**Reply by robinkwilson** [reply]
You're correct, after deleting parameter store you're back to defaults. The second part is completely optional.

Though if you were messing with that setting, I assumed you'd still want to add something to it.

**Reply by angelcabrera-nr** [reply]
Thank you @robinkwilson this information is life saving 🙌
